pkg/kv/store.go

```diff
@@ -146,7 +146,7 @@ func UnregisterAllDrivers() {
 	}
 }
 
-// Open lookup driver with 'name' and return Store based on 'dsn' (data source name).
+// Open lookup driver with 'name' and return store based on 'dsn' (data source name).
```

Comment:
If you already modified it - you can remove the DSN reference as it is irrelevant